config/config_string.go

```diff
@@ -148,6 +148,42 @@ func outputsStr(os []*Output) string {
 	return strings.TrimSpace(result)
 }
 
+func localsStr(ls []*Local) string {
```

Comment:
should this be in a test file, or is it intended to be used elsewhere eventually?